idl/VideoCapturer.idl

```diff
@@ -139,6 +139,11 @@ namespace org
       /// </summary>
       bool enableMrc;
 
+      /// <summary>
+      /// If Mixed Reality Capture is enabled, enable the recording indicator on screen.
+      /// </summary>
+      bool enableMrcRecordingIndicator;
```

Comment:
>bool enableMrcRecordingIndicator; [](start = 6, length = 33)

This is a breaking change that breaks backward compatibility. I guess this is how things roll on m71, but I'll most likely change this policy in the future. WebRTC-UWP should have a versioned API that keeps the investments of app developers.

Reply:
Right, although this is fairly limited since the default value (`true`) is the one implicitly used currently. I don't know of any versioning on the API anyway, if you find something let me know.